docs/dev/your_first_pr.md

```diff
@@ -0,0 +1,251 @@
+# Your First PR!
+
+Thanks for choosing to help improve the Tanzu Framework! This guide will
+help you build your first successful pull request.
+
+# Opening an Issue
+
+Your new idea starts with a new GitHub issue. You'll link your pull request to
+your issue once you're ready to submit it. This will help us appropriately
+prioritize the release into which your PR should get merged into.
+
+[Click here](https://github.com/vmware-tanzu/tanzu-framework/issues/new) to
+open your first issue.
+
+# Getting the Code
+
+Now that you've created your issue, your next step is to fork the repo.
+
+Your pull request will merge your fork into our repository.
+
+Click on the "Fork" button at the top right of the page to fork
+Tanzu Framework into your Github account.
+
+✅ **NOTE**: You'll need your terminal for the steps below.
+
+# Preparing Your Development Environment
+
+Next, we'll need to prepare your development environment.
+
+Before doing anything, go ahead and clone your fork with `git clone`.
+
+## Download and install Golang and GVM
+
+Tanzu Kubernetes Grid is written in [Golang](https://golang.org), or Go
+for short. While it is currently built against Go v1.1.7, this might change
```

Comment:
Transposed that Go version. I think we're using go v1.17 right now. Or even still v1.16.